### DISCUSSION TITLE
Draft replies to issues.

### DISCUSSION THREAD

**Discussion by nicolasnoble** [community/community#40580] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Sometimes, users will come up with issues that are complex and require a lot of time to properly investigate and formulate an answer.

The same way pull request reviews allow you to draft replies, I'd be useful to be able to draft comments to issues, when the investigation of said issue takes a while and I'm juggling too many plates at the same time. More often than not I would start replying, then research some items to insert into the reply or make sure I am correct, then pulled into a meeting, and a Chrome restart in the meantime meant I've lost my train of thoughts in replying.

Using my email client instead to reply to issues is my current workaround, as drafts are saved automatically, but the formatting isn't nearly as good as what the actual GitHub editor can do.

**Comment by ossanna16** [comment]
Hi there @nicolasnoble and welcome to our community! Thank you for the feedback!

**Comment by suvarchal** [comment]
Just wanted to add: it is not just for replies to issues, I often see the need to draft issues so that i can illustrate the problem clearly and articulate exploration around it.
